hack/v3benchmark/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context"
+	"github.com/coreos/etcd/Godeps/_workspace/src/google.golang.org/grpc"
+	"github.com/coreos/etcd/etcdserver/etcdserverpb"
+	"github.com/rakyll/pb"
+)
+
+func main() {
```

Comment:
Also I would suggest to check https://github.com/grpc/grpc-go/blob/master/benchmark/benchmark.go